urls.py

```diff
@@ -4,11 +4,21 @@
 import apps.site.views as app_view
 from apps.site.views import login_view
 from apps.moz_users.views import upload
+from django.conf import settings
 
-urlpatterns = [
-    url(r'^user/', include('apps.moz_users.urls')),
-    url(r"^login[/]", login_view, name="login"),
-    url(r'^$', upload),
-    url(r'^administration/', include('apps.moz_desktop.urls')),
-    url(r'^admin/', include(admin.site.urls)),
-]
+if settings.ALLOW_ADMIN:
+    urlpatterns = [
+        url(r'^user/', include('apps.moz_users.urls')),
+        url(r"^login[/]", login_view, name="login"),
+        url(r'^oidc/', include('mozilla_django_oidc.urls')),
+        url(r'^$', upload),
+        url(r'^administration/', include('apps.moz_desktop.urls')),
+        url(r'^admin/', include(admin.site.urls)),
+    ]
+else:
+    urlpatterns = [
+        url(r'^user/', include('apps.moz_users.urls')),
+        url(r"^login[/]", login_view, name="login"),
+        url(r'^oidc/', include('mozilla_django_oidc.urls')),
+        url(r'^$', upload),
+    ]
```

Comment:
Instead of copying large chucks of `urlpatterns`, it should be `urlpatterns = [...]` and then if `settings.ALLOW_ADMIN` is `True`, it should `+=` the other two. It helps to make it clearer that they are functionally the same, except for `settings.ALLOW_ADMIN` adds two new endpoints.